lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -40,6 +46,54 @@
 FEATURES_WITH_SSL_AUTH['AUTH_USE_CERTIFICATES'] = True
 
 
+class TestReportSending(SharedModuleStoreTestCase):
+    """
+    Check the functionality of sending reports.
+    """
+    @classmethod
+    def setUpClass(cls):
+        super(TestReportSending, cls).setUpClass()
+        cls.store = modulestore()
+        cls.first_lib = LibraryFactory.create(
+            org="test", library="lib1", display_name="run1", default_store=ModuleStoreEnum.Type.split
+        )
+        cls.second_lib = LibraryFactory.create(
+            org="test", library="lib2", display_name="run2", default_store=ModuleStoreEnum.Type.split
+        )
+
+        cls.first_course = CourseFactory.create(
+            org="test", course="course1", display_name="run1"
+        )
+        cls.second_course = CourseFactory.create(
+            org="test", course="course2", display_name="run1"
+        )
+        EmailsAddressMailing.objects.create(email='mail+1@exemple.com', active=True, comment='comment')
+        EmailsAddressMailing.objects.create(email='mail+2@exemple.com', active=True, comment='comment')
+        EmailsAddressMailing.objects.create(email='mail+3@exemple.com', active=False, comment='comment')
+
+    def mock_return_xls(header, data):
+        return data
+
+    @mock.patch('dashboard.tasks.return_xls', side_effect=mock_return_xls)
+    def test_get_courses_xls(self, return_xls):
+        """Assert the count of lines in the file with the count of courses."""
+        data = get_courses_xls()
+        count_courses = 2
+        self.assertEqual(count_courses, len(data))
+
+    @mock.patch('dashboard.tasks.EmailMessage')
+    @mock.patch('dashboard.tasks.get_connection')
+    @mock.patch('dashboard.tasks.settings')
+    @mock.patch('dashboard.tasks.return_xls')
+    def test_mass_sending_report(self, return_xls, settings, get_connection, EmailMessage):
+        """Assert count of active emails in the database with the count of emails to send."""
+        settings.EMAIL_FROM='mail@exemple.com'
```

Comment:
`from django.test.utils import override_settings` use this please

Reply:
https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.override_settings